common/src/main/java/org/wso2/testgrid/common/DashboardSetup.java

```diff
@@ -71,6 +76,7 @@ public void initDashboard() {
             String dbName = testplanID;
             influxDB.createDatabase(dbName);
             influxDB.close();
```

Comment:
This is incorrect usage of resource closing. Please see try-with-resource in java7 and try-finally block usage.